File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject15.scala

package org.openapitools.client.model


case class InlineObject15 (
    /* End of the time range, specified in milliseconds UNIX time. */
    _endMs: Integer,
    /* Group ID to query. */
    _groupId: Long,
    /* Beginning of the time range, specified in milliseconds UNIX time. Limited to a 90 day window with respect to startMs and endMs */
    _startMs: Integer,
    /* Vehicle ID to query. */
    _vehicleId: Long
)
object InlineObject15 {
    def toStringBody(var_endMs: Object, var_groupId: Object, var_startMs: Object, var_vehicleId: Object) =
        s"""
        | {
        | "endMs":$var_endMs,"groupId":$var_groupId,"startMs":$var_startMs,"vehicleId":$var_vehicleId
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineObject15.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param endMs End of the time range, specified in milliseconds UNIX time.
 * @param groupId Group ID to query.
 * @param startMs Beginning of the time range, specified in milliseconds UNIX time. Limited to a 90 day window with respect to startMs and endMs
 * @param vehicleId Vehicle ID to query.
 */
data class InlineObject15 (

        @get:NotNull 
        @JsonProperty("endMs") val endMs: Int,

        @get:NotNull 
        @JsonProperty("groupId") val groupId: Long,

        @get:NotNull 
        @JsonProperty("startMs") val startMs: Int,

        @get:NotNull 
        @JsonProperty("vehicleId") val vehicleId: Long
) {

}

